test/views/ChoiceViewSpec.scala

```diff
@@ -75,10 +75,10 @@ class ChoiceViewSpec extends ViewSpec with ChoiceMessages with CommonMessages {
       val view = createView(Choice.form())
 
       getElementByCss(view, "#choice>div:nth-child(2)>label").text() must be(messages(arrivalDecLabel))
-      getElementByCss(view, "#choice>div:nth-child(3)>label").text() must be(messages(departureDecLabel))
-      getElementByCss(view, "#choice>div:nth-child(4)>label").text() must be(messages(associateDecLabel))
-      getElementByCss(view, "#choice>div:nth-child(5)>label").text() must be(messages(disassociateDecLabel))
-      getElementByCss(view, "#choice>div:nth-child(6)>label").text() must be(messages(shutMucrLabel))
+      getElementByCss(view, "#choice>div:nth-child(3)>label").text() must be(messages(associateDecLabel))
```

Comment:
It can be better, as I said before, I want to avoid refactors that are not necessary for now. I'll create a list with comments to the tests etc and after ticket when we will have time I'll fix it.